packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -469,14 +464,14 @@ function validateCfnTemplates(context, resourcesToBeUpdated) {
   }
 }
 
-async function packageResources(context, resources) {
+async function packageResources(context: $TSContext, resources: $TSAny[]) {
   // Only build and package resources which are required
   resources = resources.filter(resource => resource.build);
 
   let log = null;
 
-  const packageResource = async (context, resource) => {
-    let result;
+  const packageResource = async (context: $TSContext, resource: $TSAny) => {
```

Comment:
I know it's not exactly part of this PR, but I'd recommend moving this function outside of packageResources as a non-exported function in the module. It just seems a little bit large to be a local function